lib/player/Controls.js

```diff
@@ -23,11 +23,27 @@ const PlayerProfiles = {
  * The Eluvio Player has a full set of APIs to control the video and retrieve information about the current state. Access player controls via `player.controls`
  */
 class PlayerControls {
+  static ID_SEEKBAR = "seekbar";
```

Comment:
I think we should move these to PlayerParameters.js, since they are a part of the API. something like this
```
export const ElvPlayerControlIds = {
  seekbar: "seekbar"
  ...
};
```

Additionally, add a method here called GetPlayerControl that is basically like
```
GetPlayerControl(id) {
  return document.querySelector(`#${id}-${this.player.playerId}`);
}
```

Usage: 
```
import {ElvPlayerControlIds} from "@eluvio/elv-player-js";
...
const seekbar = player.controls.GetPlayerControl(ElvPlayerControlIds.seekbar);
```